### PR TITLE
StyleCop cleaned OpenRA.Game Part 1

### PR DESCRIPTION
**Pull request by Mailaender**
Followup of https://github.com/OpenRA/OpenRA/pull/7247. It grew big sadly so I split it in two commits.

### CODE REVIEW

**Comment by Mailaender** [comment]
I had no idea what most of the code from Blast.cs and Blowfish.cs does.

**Comment by Mailaender** [comment]
Updated to avoid the large scale renamings such as OrderManager, Sprite and ModData as requested by @pchote in IRC.

File: OpenRA.Game/Graphics/VoxelRenderer.cs [hunk 1]
```diff
@@ -18,14 +18,14 @@ namespace OpenRA.Graphics
 {
 	public class VoxelRenderProxy
 	{
-		public readonly Sprite Sprite;
+		public readonly Sprite sprite;
```
Comment (RoosterDragon) [inline OpenRA.Game/Graphics/VoxelRenderer.cs]: Pretty sure this should remain uppercase.

File: OpenRA.Game/Map/Map.cs [hunk 2]
```diff
@@ -847,7 +847,7 @@ public WRange DistanceToEdge(WPos pos, WVec dir)
 			return new WRange(Math.Min(x, y) * dir.Length);
 		}
 
-		static CVec[][] TilesByDistance = InitTilesByDistance(MaxTilesInCircleRange);
+		static CVec[][] tilesByDistance = InitTilesByDistance(MaxTilesInCircleRange);
```
Comment (RoosterDragon) [inline OpenRA.Game/Map/Map.cs]: I think this would be better readonly (which then requires uppercase again).

File: OpenRA.Game/MiniYaml.cs [hunk 3]
```diff
@@ -83,19 +83,19 @@ public override string ToString()
 
 	public class MiniYaml
 	{
-		static Func<string, string> StringIdentity = s => s;
-		static Func<MiniYaml, MiniYaml> MiniYamlIdentity = my => my;
+		static Func<string, string> stringIdentity = s => s;
+		static Func<MiniYaml, MiniYaml> miniYamlIdentity = my => my;
```
Comment (RoosterDragon) [inline OpenRA.Game/MiniYaml.cs]: I think these would both be better as readonly (requiring uppercase names again).

**Comment by Mailaender** [comment]
Updated.

File: OpenRA.Game/WAngle.cs [hunk 4]
```diff
@@ -103,7 +103,7 @@ public static WAngle ArcTan(int y, int x, int stride)
 
 		public override string ToString() { return Angle.ToString(); }
 
-		static int[] CosineTable =
+		static int[] cosineTable =
```
Comment (RoosterDragon) [inline OpenRA.Game/WAngle.cs]: Prefer readonly here.

File: OpenRA.Game/WAngle.cs [hunk 5]
```diff
@@ -126,7 +126,7 @@ public static WAngle ArcTan(int y, int x, int stride)
 			56, 50, 43, 37, 31, 25, 18, 12, 6, 0
 		};
 
-		static int[] TanTable =
+		static int[] tanTable =
```
Comment (RoosterDragon) [inline OpenRA.Game/WAngle.cs]: Prefer readonly here.

File: OpenRA.Game/Widgets/Widget.cs [hunk 6]
```diff
@@ -23,7 +23,7 @@ public static class Ui
 
 		public static int LastTickTime = Game.RunTime;
 
-		static Stack<Widget> WindowList = new Stack<Widget>();
+		static Stack<Widget> windowList = new Stack<Widget>();
```
Comment (RoosterDragon) [inline OpenRA.Game/Widgets/Widget.cs]: Prefer readonly here.

**Comment by Mailaender** [comment]
Updated.

**Comment by RoosterDragon** [comment]
Looks alright to me.

File: OpenRA.Game/Exts.cs [hunk 7]
```diff
@@ -372,13 +372,17 @@ public static T[] MakeArray<T>(int count, Func<int, T> f)
 		{
 			var result = new T[width, height];
 			for (var i = 0; i < width; i++)
+			{
 				for (var j = 0; j < height; j++)
-					// Workaround for broken ternary operators in certain versions of mono (3.10 and  
-					// certain versions of the 3.8 series): https://bugzilla.xamarin.com/show_bug.cgi?id=23319
+				{
+					// Workaround for broken ternary operators in certain versions of mono (3.10 and certain versions of the 3.8 series): https://bugzilla.xamarin.com/show_bug.cgi?id=23319
```
Comment (obrakmann) [inline OpenRA.Game/Exts.cs]: imo this worked better as a two-line comment

**Comment by obrakmann** [comment]
:+1:

File: OpenRA.Game/Exts.cs (hunk 7 above)
Comment (Mailaender) [inline OpenRA.Game/Exts.cs]: Didn't know how else to solve this StyleCop nit. Let's hope this does not have to stay forever here.

**Pull request merged**